BinarAcademy.playground/Pages/Optional.xcplaygroundpage/Contents.swift

```diff
@@ -8,3 +8,30 @@ var greeting = "Hello, playground"
 
 var number: Double = 11.0
 var _number = Int(number)
+
+
+//Using variadic parameters
+func sayHello(greeting: String, names: String...) {
+    for name in names {
+        print("\(greeting) \(name)")
+    }
+}
+
+
+sayHello(greeting:"Hello", names: "Jon", "Kara")
+
+//Hello Jon
+//Hello Kara
+
+
+//inout parameters
+func reverse(first: inout String, second: inout String) {
+    let tmp = first
+    first = second
+    second = tmp
+}
+
+var one = "One"
+var two = "Two"
+reverse(first: &one, second: &two)
+print("one: \(one) two: \(two)")
```

Comment:
```suggestion
var one: String = "One"
var two: String = "Two"
reverse(first: &one, second: &two)
print("one: \(one) two: \(two)")
```
add explicit data type